x/gamm/pool-models/stableswap/amm.go

```diff
@@ -174,6 +174,7 @@ func solveCFMMBinarySearch(constantFunction func(osmomath.BigDec, osmomath.BigDe
 		yFinal := yReserve.Add(yIn)
 		xLowEst := osmomath.ZeroDec()
 		// we set upper bound at 2 * xReserve to accommodate negative yIns
+		// TODO: update this to be in accordance with spec (linear estimation)
```

Comment:
I believe this is now done!

Reply:
Good point! Changes are in #3067 now – removing this comment